reference/settings.v3.yml

```diff
@@ -30,146 +30,194 @@ servers:
   - description: Production API Gateway
     url: 'https://api.bigcommerce.com'
 paths:
-  '/stores/{store_hash}/v3/settings/analytics':
+  '/stores/{store_hash}/v3/settings/data-solutions/web-analytics':
     get:
-      summary: Get All Web Analytics Providers
-      description: Returns a list of web analytics providers.
+      operationId: get-web-analytics
+      tags:
+        - Web Analytics
+      summary: Get All Web Analytics
+      description: 'Returns a list of all available Web Analytics data for a specified channel. If `channel_id` is not set, the endpoint returns the analytics for the default channel.'
       parameters:
-        - $ref: '#/components/parameters/ChannelIdParam'
+        - name: store_hash
+          in: path
+          required: true
+          description: Unique identifier for a store.
+          schema:
+            type: string
+        - name: channel_id
+          in: query
+          description: 'ID of the storefront channel. Defaults to the ID of the default (global) channel.'
+          schema:
+            type: integer
       responses:
         '200':
-          description: OK
+          description: |-
+            Successful operation. 
+
+            <!-- theme: info -->
+            > #### Note:
+            > - The `200` response body returns an array of all analytics objects. View each individual analytic by clicking the dropdown next to **data**.
+            > - For Google Analytic, the response includes `tracking_code` or `property_id` depending on the `version`.
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  data:
-                    $ref: '#/components/schemas/AnalyticsProviders'
-                  meta:
-                    type: object
-      tags:
-        - Analytics
-    parameters:
-      - $ref: '#/components/parameters/store_hash'
-  '/stores/{store_hash}/v3/settings/analytics/{id}':
+                $ref: '#/components/schemas/Analytics'
+        '400':
+          description: Bad request. Authentication Required.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse400'
+  '/stores/{store_hash}/v3/settings/data-solutions/web-analytics/{id}':
     get:
-      summary: Get a Web Analytics Provider
-      description: Returns a single web analytics provider data for a default channel.
+      operationId: get-a-web-analytic
+      tags:
+        - Web Analytics
+      summary: Get a Web Analytic
+      description: |-
+        Returns a single Web Analytic for a specified channel. 
+
+        - If the `channel_id` is not set, the endpoint returns the analytic for the default channel.
+
+        - | Web analytic | ID |
+          | ------------- | -------- |
+          | Google Analytics | 1 |
+          | Visual Website Optimizer | 2 |
+          | Facebook Pixel | 3 |
```

Comment:
This is now called a Meta Pixel and not a Facebook Pixel: https://www.facebook.com/business/help/742478679120153?id=1205376682832142

We want to be consistent with whatever the API accepts or returns, so if this is what P&E calls it then it's cool. I am just throwing it out there in case someone has or is going to change this value.